### PR TITLE
Nice kotlin syntax `when` (like switch)

### PR DESCRIPTION
**Pull request by dsmith42**
Super small step towards more Kotlin-ish syntax

### CODE REVIEW

**Review by Gmiller290488 — APPROVED** [review]

File: androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt [hunk 1]
```diff
@@ -87,11 +87,10 @@ class MealsAdapter (var mealsList: List<Meal>) : RecyclerView.Adapter<MealsAdapt
     }
 
     fun getQuantityText(quantity: Int) : String {
```
Comment (kirstybutler) [inline androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt]: could we remove the space before the : please :)

Comment (KaneCheshire) [reply androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt]: Could also just do 
```
fun getQuantityText(quantity: Int) = when (quantity) {
    0 -> "#    Reserved"
    ... etc
}
```

Comment (kirstybutler) [reply androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt]: we have literally just been saying this in slack :) @dsmith42

File: androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt [hunk 2]
```diff
@@ -86,12 +86,9 @@ class MealsAdapter (var mealsList: List<Meal>) : RecyclerView.Adapter<MealsAdapt
...
-        } else if (quantity == 1) {
-            "#    ${quantity} portion remaining"
-        } else {
-            "#    Reserved" }
+    fun getQuantityText(quantity: Int) : String = when (quantity) {
+        0 -> "#    Reserved"
+        1 ->  "#    ${quantity} portion remaining"
+        else -> "#    ${quantity} portions remaining"
     }
 }
```
Comment (kirstybutler) [inline androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt:94]: 😍

**Review by kirstybutler — APPROVED** [review]

File: androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt (hunk 1 above)
Comment (dsmith42) [reply androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt]: Kotlin is pretty nice

Comment (KaneCheshire) [reply androidHackathonApp/src/main/java/com/kcc/kmmhackathon/androidHackathonApp/MealsAdapter.kt]: I really love Kotlin, especially the way constructors work

**Pull request merged**